wdn/templates_5.3/js-src/unlalert.babel.js

```diff
@@ -3,146 +3,162 @@ define([
   'jquery',
   'css!js-css/unlalert'
 ], function(WDN, $) {
-  var dataUrl = 'https://alert.unl.edu/json/unlcap.js';
-  var activeIds = [], calltimeout,
-
-  ckPrfx = 'unlAlerts',
-  idPrfx = 'unlalert',
-  cntSuf = '_content',
-  togSuf = '_toggle',
-  icnSuf = '_icon',
-  axnSuf = '_action',
-
-  timeoutPeriod = 30, // how ofter to check for expired data in seconds
-  dataLifetime = 30, // seconds until the data cookie expires
-  ackLifetime = 3600, // seconds until an acknowledgment expires
-
-  _getClosedAlerts = function() {
-    var c = WDN.getCookie(ckPrfx + 'C');
-    if (c) {
-      return c.split(',');
+  var alertDataUrl = 'https://alert.unl.edu/json/unlcap.js';
+  var alertDataActiveIDs = [], 
+  serverCallTimeoutID,
+  
+  closedAlertCookieName = 'unl_alerts_closed_alerts',
```

Comment:
Keep cookie names the same

Reply:
Changed cookie names to their original values. 

closedAlertCookieName = 'unlAlertsC',
 alertDataReceivedCookieName = 'unlAlertsData',
 activeAlertCookieName = 'unlAlertsA',